### PR TITLE
chore: clean some trivial code

### PR DESCRIPTION
**Pull request by muchvo**
## Description

1. Clean some trivial code
2. Update examples

## Types of changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds core functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation (update in the documentation)

## Implemented Tasks

- [ ] Subtask 1
- [ ] Subtask 2
- [ ] Subtask 3

## Checklist

Go over all the following points, and put an `x` in all the boxes that apply.
If you are unsure about any of these, don't hesitate to ask. We are here to help!

- [x] I have read the [CONTRIBUTION](https://github.com/OmniSafeAI/omnisafe/blob/HEAD/CONTRIBUTING.md) guide. (**required**)
- [ ] My change requires a change to the documentation.
- [ ] I have updated the tests accordingly. (*required for a bug fix or a new feature*)
- [ ] I have updated the documentation accordingly.
- [x] I have reformatted the code using `make format`. (**required**)
- [x] I have checked the code using `make lint`. (**required**)
- [x] I have ensured `make test` pass. (**required**)

### CODE REVIEW

File: omnisafe/utils/exp_grid_tools.py [hunk 1]
```diff
@@ -49,3 +54,45 @@ def valid_str(vals: list | str) -> str:
     str_v = str(vals).lower()
     valid_chars = f'-_{string.ascii_letters}{string.digits}'
     return ''.join(c if c in valid_chars else '-' for c in str_v)
+
+
+def train(
+    exp_id: str,
+    algo: str,
+    env_id: str,
+    custom_cfgs: NamedTuple,
```
Comment (Gaiejj) [inline omnisafe/utils/exp_grid_tools.py:63]:
```suggestion
from omnisafe.typing import Config
custom_cfgs: Config
```
I will fix it in my ``mypy`` PR #221. I just mark it but do not request change.

**Review by Gaiejj — APPROVED** [review]
LGTM.

File: omnisafe/utils/exp_grid_tools.py (hunk 1 above)
Comment (muchvo) [reply omnisafe/utils/exp_grid_tools.py:63]: got it

**Comment by zmsn-2077** [comment]
LGTM.

**Review by zmsn-2077 — APPROVED** [review]

**Pull request merged**